plugins/modules/zos_job_submit.py

```diff
@@ -310,6 +310,12 @@
         Integer of the position within the job queue where this jobs resided.
       type: int
       sample: 3
+    program_name:
```

Comment:
See my comment for program_name for zos_job_query

Reply:
I'm mirroring changes between zos_job_output, query and submit.